events/wc_offmap_events.txt

```diff
@@ -958,4 +958,326 @@ character_event = {
+				modifier = {
+					factor = 0
+					NOT = { trait = disfigured }
+				}
+			}
+			10 = {
+				limit = { trait = blinded }
+				remove_trait = blinded
+				random = {
+					chance = 60
+					add_trait = eyes_mutation
+					hidden_effect = { change_variable = { which = void_mutations value = 1 } }
+					modifier = {
+							factor = 0
+							AND = {
+								trait = void_being 
+								NOT = { trait = creature_naga }
+							}
+					}
+					modifier = {
+							factor = 0
+							can_get_mutation_healing_trigger = yes
+						}
+					modifier = {
+						factor = 0
+						trait = eyes_mutation
+					}
+				}
+				modifier = {
+					factor = 0
+					NOT = { trait = blinded }
+				}
+			}
+			10 = {
+				limit = { trait = infirm }
+				remove_trait = infirm 
+				modifier = {
+					factor = 0
+					NOT = { trait = infirm }
+				}
+			}
+			10 = {
+				limit = { trait = inbred }
+				remove_trait = inbred
+				modifier = {
+					factor = 0
+					NOT = { trait = inbred }
+				}
+			}
+			10 = {
+				limit = { trait = clubfooted }
+				remove_trait = clubfooted
+				modifier = {
+					factor = 0
+					NOT = { trait = clubfooted }
+				}
+			}
+			10 = {
+				limit = { trait = hunchback }
+				remove_trait = hunchback
+				modifier = {
+					factor = 0
+					NOT = { trait = hunchback }
+				}
+			}
+			10 = {
+				limit = { trait = dwarf }
+				remove_trait = dwarf
+				modifier = {
+					factor = 0
+					NOT = { trait = dwarf }
+				}
+			}
+			10 = {
+				limit = { trait = dwarf }
+				remove_trait = dwarf
+				modifier = {
+					factor = 0
+					NOT = { trait = dwarf }
+				}
+			}
+			10 = {
+				limit = { trait = eunuch }
+				remove_trait = eunuch
+				modifier = {
+					factor = 0
+					NOT = { trait = eunuch }
+				}
+			}
+			10 = {
+				limit = { trait = harelip }
+				remove_trait = harelip
+				modifier = {
+					factor = 0
+					NOT = { trait = harelip }
+				}
+			}
+			10 = {
+				limit = { trait = stutter }
+				remove_trait = stutter
+				modifier = {
+					factor = 0
+					NOT = { trait = stutter }
+				}
+			}
+			10 = {
+				limit = { trait = lisp }
+				remove_trait = lisp
+				modifier = {
+					factor = 0
+					NOT = { trait = lisp }
+				}
+			}
+			10 = {
+				limit = { trait = incapable }
+				remove_trait = incapable
+				modifier = {
+					factor = 0
+					NOT = { trait = incapable }
+				}
+			}
+		}
+		hidden_effect = {
+			random_list = {
+				50 = {
+					character_event = { id = WCVOD.300 }
+					modifier = {
+						factor = 0
+						can_get_mutation_healing_trigger = yes
+					}
+				}
+				50 = {}
+			}
+		}
+	}
+	after = {
+		if = {
+			limit = {
+				NOR = {
+					has_disease_trigger = yes
+					has_symptom_trigger = yes
+					has_injury_trigger = yes
+					is_maimed_trigger = yes
+					has_disability_trigger = yes
+				}
+			}
+			remove_symptoms_effect = yes
+			clear_disease_level_effect = yes
+			remove_disease_flags_effect = yes
+			hidden_effect = { remove_treatment_modifiers_effect = yes }
```

Comment:
`remove_treatment_modifiers_effect` and `clear_disease_level_effect ` should be cleared when you cure disease, not here. It's confusing.
P.S. Other effects not needed at all.